### DISCUSSION TITLE
Tried nova 2 ea, back to Nova - Feedback

### DISCUSSION THREAD

**Discussion by alexbainbridge** [deepgram/community#364] [General help] [answered]
I tried to use Nova-2-ea, in particular webstreaming / JavaScript recording / iPhone webview / iPhone 14 microphone about 40-50 cm away from mouth

But 
- too many extra words invented that were not said at all (not wrong words, just words when there were spaces)
- have to talk loudly

With exactly the same tech (hardware, code etc), moving back to Nova model
- I can talk softly, and it works
- I do see some words are not as nicely handled - e.g. We have a place in the UK called Southampton - Nova-2 handles Southampton, but Nova has "South Hampton"

So Nova 2 looks great on quality, but needs work on low volume words / not inventing words where nothing was spoken. Hope to move to Nova2 again in the future!

**Comment by jpvajda** [comment, marked as answer]
@alexbainbridge  thanks for more useful feedback on Nova-2. I'll pass this along to one of our Product Teams for evaluation.
